README.md

```diff
@@ -1 +1,96 @@
-Keep your branches clean and nice. Use flake8 and tests. Good luck!
```

Comment:
where are the tests tho? 
u wont fool me
https://fastapi.tiangolo.com/tutorial/testing/